ocs_ci/ocs/amq.py

```diff
@@ -489,27 +653,47 @@ def setup_amq_cluster(self, sc_name, namespace=constants.AMQ_NAMESPACE, size=100
         self.amq_is_setup = True
         return self
 
-    def cleanup(self, namespace=constants.AMQ_NAMESPACE):
+    def cleanup(self, kafka_namespace=constants.AMQ_NAMESPACE, tiller_namespace='tiller'):
```

Comment:
Please create a constant for triller in constants.py and use the constant here

Reply:
As discussed, moved at the top of file